tests/jest/aggregate-messages.test.js

```diff
@@ -18,22 +16,22 @@ const expectedMessages = {
 
 describe('aggregates translations messages', () => {
   it('returns empty messages when no translations directories are provided', () => {
-    const messages = aggregateMessages([], locales, fileSystem);
+    const messages = aggregateMessages([], locales);
```

Comment:
The aggregateMessages functions doesn't accept a fileSystem parameter so this was dead-test code.